src/Components/ExperimentOne/ExperimentOneDisplayInformation.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import BoredAPICaller from "../../ApiTemplates/BoredAPICaller.js";
+import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
+
+const ExperimentOneDisplayInformation = (props) => {
+  const [userName, setUserName] = useState("");
+  const roboHashUrl = "https://robohash.org/" + userName + ".png?size=200x200";
+
+  const [submitButtonState, setSubmitButtonState] = useState(false);
+  const [userActivityType, setUserActivityType] = useState("social");
+  const [userActivityDescription, setUserActivityDescription] = useState(
+    "Stare at wall"
+  );
+
+  const userOutputMessage =
```

Comment:
Take a look at String Template Literals... they will help to clean this up

Reply:
String literals are amazing! Thanks for letting me know